packages/SwingSet/tools/paths.js

```diff
@@ -3,9 +3,14 @@ import { resolve } from 'import-meta-resolve';
 
 /**
  * @param {string} specifier
+ * @param {string} base
  * @returns {string} file system absolute path of the package resource specifier
  */
-export const pkgAbsPath = specifier => {
-  const href = resolve(specifier, import.meta.url);
+export const resolvePath = (specifier, base) => {
+  const href = resolve(specifier, base);
   return url.fileURLToPath(href);
 };
+
+export const makeResolvePath = base => {
+  return specifier => resolvePath(specifier, base);
+};
```

Comment:
Alright, this is a good compromise.

@warner I would encourage us to in general migrate as much of our API surface to treating file URLs as first-class, starting with `@endo/bundle-source` and working our way out. `@endo/compartment-mapper` APIs are already framed in terms of file URLs.